README.md

```diff
@@ -1,8 +1,12 @@
-Buck
-====
-
-Buck is a build tool. To see what Buck can do for you,
-check out the documentation at <http://buck.build/>.
+<p align="center">
+  <img src="https://buck.build/static/og.png" alt="logo" width="20%" />
+</p>
+<h1 align="center">
+  Buck
+</h1>
+<p align="center">
+  Buck is a build tool. To see what Buck can do for you, check out the documentation at <http://buck.build/>.
```

Comment:
![Screen Shot 2020-08-14 at 6 01 15 AM](https://user-images.githubusercontent.com/50673687/90252504-79983b80-ddf4-11ea-816b-70508752d48f.png)
@Saifulriza123 Could you check why the documentation link is missing in the screenshot? Thank you for your contributions.